docs/plugins/Verifier/cosign.md

```diff
@@ -1,23 +1,31 @@
+---
+sidebar_position: 2
+---
+
 # Cosign
 
-This README outlines how this validation framework can be used to verify signatures generated using [cosign](https://github.com/sigstore/cosign/). The verifier is added as a plugin to the framework that uses [cosign](https://github.com/sigstore/cosign/) packages to invoke the verification of an image. Cosign verifier works with remote registry that can provide cosign related artifacts linked as specially formatted tag to the subject artifact. It also is compatible with OCI 1.1 supported Cosign which pushes the signature OCI Image as a referrer to the subject image. (Note: this is currently experimental for cosign) It works only with [oras](../../reference/crds/stores.md#oras) referrer store plugin that uses the OCI registry API to discover and fetch the artifacts.
+This README outlines how this validation framework can be used to verify signatures generated using [cosign](https://github.com/sigstore/cosign/). The verifier is added as a plugin to the framework that uses [cosign](https://github.com/sigstore/cosign/) packages to invoke the verification of an image. Cosign verifier works with remote registry that can provide cosign related artifacts linked as specially formatted tag to the subject artifact. It also is compatible with OCI 1.1 supported Cosign which pushes the signature OCI Image as a referrer to the subject image. (Note: this is currently experimental for cosign) It works only with [oras](../Store/oras.md) referrer store plugin that uses the OCI registry API to discover and fetch the artifacts.
 
 ## Fallback in OCIRegistry store
+
 A configuration flag called `cosignEnabled` is introduced to the plugin configuration. If this flag is enabled, the `ListReferrers` API will attempt to query for the cosign signatures for a subject in addition to the references queried using `referrers API`. If `cosignEnabled` is `false`, then only OCI 1.1 compatible Cosign signatures will be considered. All the cosign signatures are returned as the reference artifacts with the artifact type `application/vnd.dev.cosign.artifact.sig.v1+json` This option will enable to verify cosign signatures against any registry including the ones that don't support the [notaryproject](https://github.com/notaryproject)'s `referrers` API.
 
-# Signing
+## Signing
+
 Please refer cosign documentation on how to sign an image using cosign using [key-pair based signatures](https://docs.sigstore.dev/key_management/signing_with_self-managed_keys/) and [keyless signatures](https://docs.sigstore.dev/signing/quickstart/#keyless-signing-of-a-container).
 
-# Verification
+## Verification
+
+### Key-pair based verification
```

Comment:
should we mention the new trust policy and KMP reference in this section?

Reply:
yes I plan to add this. thanks for reminder